app/models/setting.rb

```diff
@@ -10,9 +10,42 @@ class Setting < ApplicationRecord
 
   TYPES = %w{integer boolean hash array string}
   FROZEN_ATTRS = %w{name category full_name}
-  NONZERO_ATTRS = %w{puppet_interval idle_timeout entries_per_page max_trend outofsync_interval}
-  BLANK_ATTRS = %w{ host_owner trusted_hosts login_delegation_logout_url authorize_login_delegation_auth_source_user_autocreate root_pass default_location default_organization websockets_ssl_key websockets_ssl_cert oauth_consumer_key oauth_consumer_secret login_text
-                    smtp_address smtp_domain smtp_user_name smtp_password smtp_openssl_verify_mode smtp_authentication sendmail_arguments sendmail_location http_proxy http_proxy_except_list ssl_certificate ssl_ca_file ssl_priv_key }
+  NONZERO_ATTRS = %w{
```

Comment:
I would prefer leaving whitespace changes outside of this PR (perhaps additional Refs commit could work), to keep the amount of changes directly related to the issue at minimum.

Reply:
I agree in general, but every edit of that huge line is a git conflict anyway... Breaking it up is a git conflict too, but for the last time. It can be confusing tho. I can remove that.